rest/route/patch_test.go

```diff
@@ -865,3 +867,263 @@ buildvariants:
 	assert.NoError(t, err)
 	assert.Len(t, tasks, 4)
 }
+
+func TestSchedulePatchActivatesInactiveTasks(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	var generatedProject = []string{`
```

Comment:
nit: there's no need to declare this as a `var` since `generatedProject := []string{...` is short-hand for `var generatedProject = []string{...` anyways. Was this supposed to be a `const`?

Reply:
Yes it was, but turns out that can't be done for an array. Changed to `:=` notation